### PR TITLE
doc: Improved general domain model documentation

### PR DESCRIPTION
**Pull request by czkum**
- Improved ambiguous description and images.
- Used UML diagrams for description.
- Added diagram source as an XMI 1.1 export.

### CODE REVIEW

**Comment by czkum** [comment]
In file images/inspectionPlanAndMeasurementsExample.png the measured value 1.1 references a part. Rather it should point to a characteristic.

**Review by czkum — COMMENTED** [review]
In file images/inspectionPlanAndMeasurementsExample.png the measured value 1.1 references a part. Rather it should point to a characteristic.

**Comment by czthiele** [comment]
In file images/inspectionPlanAndMeasurementsExample.png the number of the two measurements need to be switched so that they match the parts

File: _pages/API/01-general.md [hunk 1]
```diff
@@ -68,122 +68,85 @@ This is the interface provided directly by PiWeb Server. It consists of two main
...
+  A limit that determines when a warning about bad values should be triggered
 
- An example would be the machine catalog. This contains the different machines which could be used to measure a part. Additional information for each machine can be noted there too. Now an inspector selects the responsible machine from the catalog when entering a measurement, which makes things a lot easier. Other values than those stored in the catalog are not possible. Also simple variants, like the direction catalog, which contains directions like left and right, are possible.
-<hr>
-### Additional Data
-Additional data are attachments that can be added to any entity in the inspection plan. This can be text, images, log files, CAD models or any binary file. There is no limit to the number of files and you can edit or remove additional data as desired.
+### Configuration
+In the PiWeb configuration attributes can be defined ready to be used at parts, characteristics, measurements, measured values and catalog entries. The available catalogs and their contained catalog entries are specified as well.<br>
+<br>
+<img src="/PiWeb-Api/images/configuration.png" class="img-responsive center-block">
```
Comment (czthiele) [inline _pages/API/01-general.md:149]: Event though the catalog is shown in the image, the description was removed completely. Is that intentional? I think there should at least be a short description on what a catalog actually is and how it is used.

**Review by czthiele — APPROVED** [review]
Other than the small changes requested, nice work! The diagrams look consistent, thumbs up for adding the .xml export for futire edits.

**Comment by czkum** [comment]
> 
Fixed.

File: _pages/API/01-general.md (hunk 1 above)
Comment (czkum) [reply _pages/API/01-general.md:149]: Removing the description was done by mistake. I've re-added the text. Well spotted!

**Comment by czkum** [comment]
> In file images/inspectionPlanAndMeasurementsExample.png the number of the two measurements need to be switched so that they match the parts

Done.

**Comment by czkum** [comment]
> In file images/inspectionPlanAndMeasurementsExample.png the measured value 1.1 references a part. Rather it should point to a characteristic.

Done.

**Pull request merged**